internal/storage/entry.go

```diff
@@ -243,11 +251,11 @@ func (s *Storage) cleanupEntries(feedID int64, entryHashes []string) error {
 		DELETE FROM
 			entries
 		WHERE
-			feed_id=$1
-		AND
-			id IN (SELECT id FROM entries WHERE feed_id=$2 AND status=$3 AND NOT (hash=ANY($4)))
+			feed_id=$1 AND
+			status=$3 AND
+			NOT (hash=ANY($4))
```

Comment:
```suggestion
			status=$2 AND
			NOT (hash=ANY($3))
```

Reply:
`NOT (hash=ANY($3))` could also be rewritten to `hash NOT IN $3`

Reply:
I'm not sure how go handles postgresql's arrays vs. lists, so I didn't change it, see [here](https://stackoverflow.com/questions/34627026/in-vs-any-operator-in-postgresql) for a deeper explanation